db_stress_tool/db_stress_test_base.cc

```diff
@@ -2771,7 +2789,9 @@ void StressTest::Reopen(ThreadState* thread) {
           clock_->TimeToString(now / 1000000).c_str(), num_times_reopened_);
   Open(thread->shared);
 
-  if ((FLAGS_sync_fault_injection || FLAGS_disable_wal) && IsStateTracked()) {
+  if ((FLAGS_sync_fault_injection || FLAGS_disable_wal ||
+       FLAGS_manual_wal_flush_one_in > 0) &&
```

Comment:
Should we also do this in reopen?? I did this and turned up CI jobs which founds quite a bit verification error.

Reply:
Debugged and fixed by disabling features can't run under unsync data loss like file ingestion.